src/EditMode.js

```diff
@@ -13,7 +13,7 @@ import * as Text from "./Text.js";
  * @param {string} meiFile - The path to the MEi file.
  * @param {module:Zoom~ZoomHandler} zoomHandler - The ZoomHandler object.
  */
-function EditMode (neonView, meiFile, zoomHandler){
+function EditMode (neonView, neon, meiFile, zoomHandler){
```

Comment:
Update jsdoc constructor to include Neon (or is it NeonView?)

Reply:
Generally we've tried to avoid passing the Neon object to other classes or functions, and have instead used NeonView as an intermediary. Doesn't matter if we actually need Neon for some reason.

Also if it is a NeonView parameter, it's clearer to name it `view` or `neonView` rather than `neon` at least in my opinion.